data/categories.txt

```diff
@@ -100,7 +100,7 @@ sk:Doprava
 @fuel
 en:2Gas
 ru:3^Заправка
-ar:1الغاز
+ar:1وقود
```

Comment:
Is it true that typing one letter is enough to start suggesting "Fuel" here? If not, please remove the '1'.